Makefile

```diff
@@ -179,6 +179,12 @@ proto: docker-build-proto
 	| docker run -i pachyderm_proto \
 	| tar xf -
 
+# Use this to grab a binary for profiling purposes
+grab-binary: docker-clean-pachd docker-build-compile
```

Comment:
Ok, thats a good point about verbs, but `make pachd-binary` sounds too generic for the use thats intended here ... it seems like I should use that make task somehow when building pachd images.

How about `make pachd-profiling-binary` instead?